controllers/drplacementcontrol_controller.go

```diff
@@ -1759,3 +1761,30 @@ func addOrUpdateCondition(conditions *[]metav1.Condition, conditionType string,
 
 	return false
 }
+
+// Initial creation of the DRPC status condition. This will also preserve the ordering of conditions in the array
+func ensureDRPCConditionsInited(conditions *[]metav1.Condition, observedGeneration int64, message string) {
```

Comment:
any reason to accept a separate observedGeneration and message? Since this is used only
at initialization, we always use `drpc.Generation` and "Initialization". We could accept a single argument: `drpc` and use its fields. This makes the function impossible to use incorrectly. Do we really the need to flexibility to call with any generation or message?